packages/fortress/views/footer.ejs

```diff
@@ -0,0 +1,3 @@
+<script src="/fortress.js"></script>
+</body>
+</html>
```

Comment:
I think the more typical approach for EJS projects is to organize files in `/pages` for pages and `/partials` for pieces of pages, but this OK for this app. `footer`/`header` makes me think of components and not layout-related pieces but Googling around shows me a lot of small projects don't tend to make this separation. 🤨

Reply:
if only ejs had layouts :(

Reply:
Not important for this PR, but we could do something like layouts with an express wrapper like https://www.npmjs.com/package/express-ejs-layouts

(it's basically how we do layouts with handlebars in email senders)